contracts/Nexus.sol

```diff
@@ -207,7 +207,6 @@ contract Nexus is INexus, EIP712, BaseAccount, ExecutionHelper, ModuleManager, U
         else if (moduleTypeId == MODULE_TYPE_EXECUTOR) _uninstallExecutor(module, deInitData);
         else if (moduleTypeId == MODULE_TYPE_FALLBACK) _uninstallFallbackHandler(module, deInitData);
         else if (moduleTypeId == MODULE_TYPE_HOOK) _uninstallHook(module, deInitData);
-        else revert UnsupportedModuleType(moduleTypeId);
```

Comment:
why is this removed?
If I pass module address which returns module typeId 5 first check will pass.
seemingly it doesn't make difference if you just return in else case but removing revert also doesn't do anything good here.